src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumModule.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.bridge.sdk;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.Deserializers;
+import com.fasterxml.jackson.databind.deser.Deserializers.Base;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+@SuppressWarnings("serial")
+class LowercaseEnumModule extends SimpleModule {
+
+    public LowercaseEnumModule() {
+        super("bridge-enum-module", new Version(1, 0, 0, "", "org.sagebionetworks.bridge", "sdk"));
+        addSerializer(Enum.class, new LowercaseEnumSerializer());
+    }
+    
+    @Override
+    public void setupModule(SetupContext context) {
+        super.setupModule(context);
+        Base deser = new Deserializers.Base() {
+            @SuppressWarnings("unchecked")
+            @Override
+            public JsonDeserializer<?> findEnumDeserializer(Class<?> type, DeserializationConfig config,
+                    BeanDescription beanDesc) throws JsonMappingException {
+                return new LowercaseEnumDeserializer((Class<Enum<?>>) type);
+            }
```

Comment:
According to the docs: "ObjectMapper: object mappers cache serializers and deserializers that are created first time handler is needed for given type (or more precisely, mapper holds a reference to Provider objects that do). If you do not reuse mappers, new serializers and deserializers need to be created each time: and these are expensive operations due to amount of introspection and annotation processing involved." 

I think this means that the results of this call are cached. Probably overkill to cache ourselves.